.dockerignore

```diff
@@ -0,0 +1,9 @@
+# Generated, non source stuff
+/.git
```

Comment:
Ezeket a dolgokat nem szeretnénk a production konténerben. Ami bekerül a konténerbe, az felkerül a publikus repositoryba is, ezért megszabadulunk mindentől ami:
* kulcs, secret
* logok és egyéb generált dolgok
* adatbázis fájlok
* fejlesztéshez használt dolgok